doc/source/user/tutorial-svd.rst

```diff
@@ -0,0 +1,468 @@
+
+To build the final approximation matrix, we must understand how multiplication
+across different axes works.
+
+Products with n-dimensional arrays
+----------------------------------
+
+If you have worked before with only one- or two-dimensional arrays in NumPy,
+you might use `numpy.dot` and `numpy.matmul` (or the ``@`` operator)
+interchangeably. However, for n-dimensional arrays, they work in very different
+ways. For more details, check the documentation `numpy.matmul`.
+
+Now, to build our approximation, we need first to make sure that our singular
+values are ready for multiplication, so we build our ``Sigma`` matrix similarly
+to what we did before. The ``Sigma`` array must have dimensions
+``(3, 768, 1024)``.
+
+::
+
+    >>> Sigma = np.zeros((3, 768, 1024))
+    >>> for j in range(3):
+    ...     for i in range(768):
+    ...         Sigma[j, i, i] = s[j, i]
+
+Now, if we wish to rebuild the full SVD (with no approximation), we could do
+
+::
+
+    >>> reconstructed = U @ Sigma @ Vt
+
+Note that
+
+::
+
+    >>> reconstructed.shape
+    (3, 768, 1024)
+
+and
+
+::
+
+    >>> plt.imshow(np.transpose(reconstructed, (1, 2, 0)))
+
+Should give you an image indistinguishable from the original one (although we
+do possibly introduce floating point errors for this reconstruction; in fact, it
+is possible that you see a warning message saying "Clipping input data to the
+valid range for imshow with RGB data ([0..1] for floats or [0..255] for
+integers)." This is expected from the manipulation we just did on the original
+image.
+
+Now, to do the approximation, we must choose only the first ``k`` singular
+values for each color channel. This can be done by the following syntax:
+
+::
+
+    >>> approx_img = U @ Sigma[..., :k] @ Vt[..., :k, :]
+
+You can see that we have selected only the first ``k`` components of the last
+axis for ``Sigma`` (this means that we have used only the first ``k`` columns
+of each of the three matrices in the stack), and that we have selected only the
+first ``k`` components in the second-to-last axis of ``Vt`` (this means we have
+selected, from every matrix in the stack ``Vt``, the first ``k`` rows only, and
+all columns). If you are unfamiliar with the ellipsis syntax, it is a
+placeholder for other axes; for more details, see the documentation on
+:ref:`Indexing <basics.indexing>`.
+
+Now,
+
+::
+
+    >>> approx_img.shape
+    (3, 768, 1024)
+
+which is not the right shape for showing the image. So, finally, reordering the
+axes back to our original shape of (768, 1024, 3), we can see our approximation:
+
+::
+
+    >>> plt.imshow(np.transpose(approx_img, (1, 2, 0)))
+
+Even though the image is not as sharp, using a small number of ``k`` singular
+values (compared to the original set of 768 values) we can recover many of the
+distinguishing features from this image.
+
+Final words
+-----------
+
+Of course, this is not the best method to *approximate* or *compress* an image.
```

Comment:
You haven't used "compress" in the rest of this tutorial. So maybe either leave out here or explain how the SVD outputs give the opportunity for better compression?